### DISCUSSION TITLE
How to make a Pole figure from EBSD data?

### DISCUSSION THREAD

**Discussion by jucego333** [mtex-toolbox/mtex#1034] [Ask Anything]
Dear colleagues

I hope you are fine. I am still new in MTex. I would like to plot a pole figure which shares the same color as EBSD. As you can see from my figures, the colors from EBSD picture are different from pole figure. I would be very grateful if somebody can give me a guidelines. Thanks

![image](https://user-images.githubusercontent.com/78469467/118322412-a61ee680-b531-11eb-85b4-8bbfc6036618.png)
![image](https://user-images.githubusercontent.com/78469467/118322515-d4042b00-b531-11eb-9b7c-70e55a2e22e3.png)

**Comment by DriptD** [comment]
Hello,

I think you want a scatter plot of the kind shown here: https://mtex-toolbox.github.io/EBSDOrientationPlots.html#3

You may try this:

```
h = [Miller(3,3,2,ebsd.CS),Miller(1,1,2,ebsd.CS)];
oM = ipfHSVKey(ebsd);
colors = oM.orientation2color(ebsd.orientations);
plotPDF(ebsd.orientations,colors,h,'all','antipodal','MarkerSize',5)
```

Regards,
Dripta

**Reply by jucego333** [reply]
Thank you so much @DriptD  it works! By the way, how can we get the texture components {hkl} <uvw>  of a given point??

**Reply by DriptD** [reply]
Hi,

You may take a look at the discussions over here: https://groups.google.com/g/mtexmail/search?q=texture%20component

There's a recent (open access) paper 'Visualization of texture components in MTEX', which I think could also be of help.

Regards,
Dripta
